tests/devworkspace-happy-path/launch.sh

```diff
@@ -67,6 +67,9 @@ function provisionOpenShiftOAuthUser() {
         }
       }
     }]'
+  #apply just added identity providers, we need to rollout deployment for make sure
```

Comment:
link to the fixing issue would help to understand which concrete problem it is fixing